.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2.1
+
+orbs:
+  ci: bigcommerce/internal@volatile
+  php: bigcommerce/internal-php@volatile
+
+jobs:
+  tests-unit:
+    parameters:
+      php-version:
+        type: string
+        default: php-7.3
```

Comment:
can you add also 7.4? we use this project on both versions.

Reply:
added.